gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/EnvironmentsResource.java

```diff
@@ -74,20 +70,18 @@ public EnvironmentResource getEnvironmentResource(
     @GET
     @Produces(io.gravitee.common.http.MediaType.APPLICATION_JSON)
     @Path("/permissions")
-    @ApiOperation(value = "List available environments with their permissions for current user organization")
-    @ApiResponses(
-        {
-            @ApiResponse(
-                code = 200,
-                message = "Current user permissions on its environments",
-                response = char[].class,
-                responseContainer = "Map"
-            ),
-            @ApiResponse(code = 500, message = "Internal server error"),
-        }
+    @Operation(summary = "List available environments with their permissions for current user organization")
+    @ApiResponse(
+        responseCode = "200",
+        description = "Current user permissions on its environments",
+        content = @Content(
+            mediaType = io.gravitee.common.http.MediaType.APPLICATION_JSON,
+            schema = @Schema(implementation = EnvironmentPermissionsList.class)
```

Comment:
why not using an `array` here ?
```suggestion
            array = @ArraySchema(schema = @Schema(implementation = EnvironmentPermissionsEntity.class))

```